components/infobox/wikis/fortnite/infobox_person_player_custom.lua

```diff
@@ -51,6 +60,7 @@ function CustomInjector:parse(id, widgets)
 	if id == 'history' then
 		local manualHistory = _args.history
 		local automatedHistory = TeamHistoryAuto._results{
+			addlpdbdata = 'true',
```

Comment:
since the rewrite of tha module some months ago we can pass along bools instead of string bools